src/core/entities/MRTextInputEntity.js

```diff
@@ -51,7 +51,8 @@ export class MRTextInputEntity extends MRTextEntity {
      * @function
      * @description Callback function of MREntity - handles setting up this textarea once it is connected to run as an entity component.
```

Comment:
can you add `(async)` to the description like you did for the other newly async functions?